### ISSUE TITLE
Cypress integration with axe-core

### ISSUE THREAD

**Issue by lucianmurmurache** [activist-org/activist#775]
### Terms

- [X] I have searched [open and closed issues](https://github.com/activist-org/activist/issues?q=is%3Aissue)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Issue

E2E tests using a mix of [cypress](https://www.npmjs.com/package/cypress) [cypress-axe](https://www.npmjs.com/package/cypress-axe) and [axe-core](https://www.npmjs.com/package/axe-core) to test accessibility violations in the frontend.

The idea is to create a spec for each endpoint and check for violations. Once this is done, we can then expand these tests to cover more testing. 

Related issue: #432

**Comment by andrewtavis** [comment]
Assigning you for this as discussed, @lucianmurmurache :)

**Comment by lucianmurmurache** [comment]
Hey @andrewtavis, unfortunately I will have to pause my contributions to activist for some time. Apologies for bowing out before finishing this issue. aasimsyed reached out and I provided some context on what's left to be done before #777 can be merged.

**Comment by andrewtavis** [comment]
Thanks for letting us know, @lucianmurmurache! Really appreciate your contributions to the project and the care you put into helping us be a more accessible platform 😊 Thank you further for sending along the information needed to finish up #777.

@aasimsyed, let's discuss in the sync what was communicated to you :) Happy to support as I can, and once we have the PR merged we can start working on making the individual issues for testing. This will be a big step for the community, with all of these issues being a great way to bring more people into the front end 😊

Looking forward to discussing later!

**Comment by andrewtavis** [comment]
Merged in #777, but we'll be switching over to Playwright based on discussions between @aasimsyed and I. Thanks for the work here, @lucianmurmurache! Looking forward to when you have a bit more time, and see you soon hopefully!
